joomla/joomla-cms
# Add release news link to the mail

Pull Request for Issue #16090

### Summary of Changes

Add release news link to the mail. cc @brianteeman & @sharphosting 

![image](https://cloud.githubusercontent.com/assets/2596554/26285464/c22aa140-3e50-11e7-801d-b95cc5e39626.png)

### Testing Instructions

- install 3.7.0
- apply this patch
- see the mail the core sends out about 3.7.1

### Expected result

Release news link is included

### Actual result

Release news link is not included

### Documentation Changes Required

none
